scripts/rhoam-manifest-generator.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Set the service name and version
+SERVICE_NAME="services-rhoam"
+
+RHOAM_VERSION=$(grep managed-api-service.v deploy/olm-catalog/managed-api-service/managed-api-service.package.yaml | tail -c 6)
+
+# Manifest folder search
+MW_MANIFESTS=( 3scale cloud-resources grafana marin3r rhsso )
```

Comment:
Can this be removed or am I missing something.

Reply:
Yes, this is part of old script, thanks!